mtools/mloginfo/sections/query_section.py

```diff
@@ -76,9 +77,9 @@ def run(self):
 
             if (le.operation in ['query', 'getmore', 'update', 'remove'] or
                     le.command in ['count', 'findandmodify',
-                                   'geonear', 'find']):
+                                   'geonear', 'find','aggregate']):
                 lt = LogTuple(namespace=le.namespace, operation=op_or_cmd(le),
-                              pattern=le.pattern, duration=le.duration)
+                              pattern=le.pattern, duration=le.duration,allowDiskUse = le.allowDiskUse)
```

Comment:
🏳️

Reply:
:D in this case 2 extra white space around the '=' and 1 missing after the comma